modules/base/rendering/screenspaceimagelocal.cpp

```diff
@@ -92,7 +92,7 @@ ScreenSpaceImageLocal::ScreenSpaceImageLocal(const ghoul::Dictionary& dictionary
 
     if (p.texturePath.has_value()) {
         if (std::filesystem::is_regular_file(absPath(*p.texturePath))) {
-            _texturePath = absPath(*p.texturePath).string();
```

Comment:
Same here. Breaking change since now the user would have to do the `openspace.absPath`call

Reply:
Made it even shorter since I didn't look at the `is_regular_file` check, which is also superfluous as the `codegen::bake` will already throw an exception.  so the `else` case here will never fire anyway